### PR TITLE
Back from the 'desistir' to beginning

### PR DESCRIPTION
**Pull request by hfcpeixoto**

### CODE REVIEW

**Comment by hfcpeixoto** [comment]
Please review this, @casey-collab

File: portugues/colabora/fork/comdiferencas/diff/pull/desisto/desistir.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 Quando você desiste você descobre que ainda precisa usar o programa que tem o bug e  
-que não tem outra opção senão conseguir arrumar o bug.
\ No newline at end of file
+que não tem outra opção senão conseguir arrumar o bug.
+
+Voltar para o [inicio](../../../../../../)
```
Comment (casey-collab) [inline portugues/colabora/fork/comdiferencas/diff/pull/desisto/desistir.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**